tests/snmpsim/ciena-saos_saos.snmprec

```diff
@@ -0,0 +1,1288 @@
+1.3.6.1.2.1.1.1.0|4|6500 7-SLOT OPTICAL SW:1271 BN:AH (c) Ciena (R) Corporation
```

Comment:
Could you generate the test data as well (using scripts/save-test-data.php) and add it here ?

Reply:
test data is added: tests/snmpsim/ciena-saos_saos.snmprec